dashboards/ebs/ebs_snapshot_detail.sp

```diff
@@ -297,7 +314,7 @@ query "aws_ebs_snapshot_relationships" {
     where
       snapshot.kms_key_id = kms_keys.arn
 
-    -- kms > vol - edges
+    -- KMS > EBS (edge)
```

Comment:
```suggestion
    -- To KMS keys (edge)
```

Reply:
@cbruno10  this is a transitive edge which connects the EBS volume of the Snapshot to the KMS key.

Changing this to: `EBS volume > KMS Key (edge)`